contrib/podmanimage/README.md

```diff
@@ -17,22 +17,21 @@ default to `/`.
 The container images are:
 
   * `quay.io/containers/podman:<version>` and `quay.io/podman/stable:<version>` -
-    These images are built when a new podman version becomes available in
+    These images are built when a new Podman version becomes available in
     Fedora.  These images are intended to be unchanging and stable, they will
-    never be updated by automation once they've been pushed.  For build
-    details, see the configuration used to build it,
-    [podmanimage/stable/Dockerfile](stable/Dockerfile).
+    never be updated by automation once they've been pushed.  For build details,
+    please [see the configuration file](stable/Dockerfile).
   * `quay.io/containers/podman:latest` and `quay.io/podman/stable:latest` -
-    Built daily using the same Containerfile as above.  The podman version
+    Built daily using the same Dockerfile as above.  The podman version
```

Comment:
sorry @cevich, missed this nit yesterday.
```suggestion
    Built daily using the same Dockerfile as above.  The Podman version
```

Reply:
woopsie-doodle.  Thanks.  Got it.